opennms-config/src/main/java/org/opennms/netmgt/config/EventdConfigManager.java

```diff
@@ -61,21 +61,21 @@
      * <p>Constructor for EventdConfigManager.</p>
      *
      * @param stream a {@link java.io.InputStream} object.
-     * @throws org.exolab.castor.xml.MarshalException if any.
-     * @throws org.exolab.castor.xml.ValidationException if any.
      * @throws java.io.IOException if any.
      */
-    public EventdConfigManager() throws MarshalException, ValidationException, IOException {
+    public EventdConfigManager() throws IOException {
         reload();
     }
     
-    EventdConfigManager(final InputStream stream) throws MarshalException, ValidationException, IOException {
-            m_config = CastorUtils.unmarshal(EventdConfiguration.class, stream);
+    EventdConfigManager(final InputStream stream) throws IOException {
+        try (final InputStreamReader isr = new InputStreamReader(stream)) {
+            m_config = JaxbUtils.unmarshal(EventdConfiguration.class, isr);
         }
+    }
     
-    private void reload() throws MarshalException, ValidationException, IOException {
-    	InputStream stream = new FileInputStream(ConfigFileConstants.getFile(ConfigFileConstants.EVENTD_CONFIG_FILE_NAME));
-    	m_config = CastorUtils.unmarshal(EventdConfiguration.class, stream);		
+    private void reload() throws IOException {
+        final Reader r = new FileReader(ConfigFileConstants.getFile(ConfigFileConstants.EVENTD_CONFIG_FILE_NAME));
```

Comment:
Shouldn't we close the FileReader after the unmarshal? We could also pass the File directly to the JaxbUtils.